src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -170,20 +170,32 @@ public class SnowflakeStreamingIngestClientInternal<T> implements SnowflakeStrea
     if (!isTestMode) {
       // Setup request builder for communication with the server side
       this.role = prop.getProperty(Constants.ROLE);
-      try {
-        KeyPair keyPair =
-            Utils.createKeyPairFromPrivateKey(
-                (PrivateKey) prop.get(SFSessionProperty.PRIVATE_KEY.getPropertyKey()));
-        this.requestBuilder =
-            new RequestBuilder(
-                accountURL,
-                prop.get(USER).toString(),
-                keyPair,
-                this.httpClient,
-                String.format("%s_%s", this.name, System.currentTimeMillis()));
-      } catch (NoSuchAlgorithmException | InvalidKeySpecException e) {
-        throw new SFException(e, ErrorCode.KEYPAIR_CREATION_FAILURE);
+
+      Object credential = null;
+      if (prop.getProperty(Constants.AUTHORIZATION_TYPE).equals(Constants.JWT)) {
```

Comment:
Could we make JWT the default authentication way so it could be backward compatible? If it's already done, could you add a comment to make it clear?

Reply:
By default the client will use JWT if it's not specified in the profile. I added some comment to make it clear.